docs/installation/cloud/index.md

```diff
@@ -0,0 +1,25 @@
+<!--[metadata]>
++++
+aliases = [
+  "/engine/installation/amazon/",
+  "/engine/installation/google/",
+  "/engine/installation/softlayer/",
+  "/engine/installation/azure/",
+  "/engine/installation/rackspace/",
+  "/engine/installation/joyent/"
+]
+title = "In the cloud"
+description = "Cloud Installations"
+keywords = ["Docker install "]
+[menu.main]
+identifier = "install_cloud"
+parent="engine_install"
+weight="-60"
++++
+<![end-metadata]-->
+
+# Install Engine in the cloud
+
+* [Understand cloud install options and choose one](cloud.md)
+* [Example: Use Docker Machine to provision cloud hosts](cloud-ex-machine-ocean.md)
+* [Example: Manual install on a cloud provider](cloud-ex-aws.md)
```

Comment:
shouldn't we just rename `cloud/cloud.md` to `cloud/index.md`, or would that require the foldable menu-items to be clickable?